### ISSUE TITLE
Texture aliasing on textures when loading from gltf using MASK

### ISSUE THREAD

**Issue by sweco-sekrsv** [CesiumGS/cesium#11401]
There is a lot of texture aliasing when loading a png texture using alphaMode = MASK from a gltf file. This is very noticeable when the camera moves. To me it looks like there is no mipmapping enabled in Cesium, so perhaps Cesium have failed to generate the mipmap?

The textures looks fine in for example babylon.js and three.js and there are no aliasing in the textures. (They look fine at a distance and not only when you are close to the model)

I have attached an example model with the gltf, glb and the texture.

[aliasing_bug.zip](https://github.com/CesiumGS/cesium/files/11979289/aliasing_bug.zip)

**Comment by sweco-sekrsv** [comment]
I looked at this some more and found that the problem seems to be linked to using KHR_texture_transform. I used gltfpack to produce the glb file. With the default options it adds KHR_texture_transform extension and this seems to break the mipmap generation in Cesium. When converting  using gltfpack with -noq (disable quantization; produces much larger glTF files with no extensions) the glb is not using KHR_texture_transform and it loads fine and the texture aliasing is gone.

Se attached glbs files for debugging this.

[glb.zip](https://github.com/CesiumGS/cesium/files/11980460/glb.zip)
